test/unit/ApiResponse/FetchedReportApiResponseTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Silktide\ProspectClient\UnitTest\ApiResponse;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+use Silktide\ProspectClient\ApiResponse\FetchedReportApiResponse;
+use Silktide\ProspectClient\Entity\Report;
+
+class FetchedReportApiResponseTest extends TestCase
+{
+    public function testGetReport()
+    {
+        $httpResponse = self::createMock(ResponseInterface::class);
```

Comment:
Talking Point: This works fine, but have you considered using anonymous' classes at all for this kind of mock?
```
$httpResponse = new class implements ResponseInterface {
   public function getStatusCode() { 
     return 200;
   }
   public function getBody() {
       return json_encode(["report" => "nothing"]);
   }
}
```

ends up feeling a bit easier to understand for me.

This isn't a request to change it, just a talking point :P